### ISSUE TITLE
det_mv3_db_v2.0训练内存不断上升，直至崩溃

### ISSUE THREAD

**Issue by xialei2821212670** [PaddlePaddle/PaddleOCR#2969]
![image](https://user-images.githubusercontent.com/57654726/120127541-388dde00-c1f2-11eb-9751-b1d53b16729b.png)

![image](https://user-images.githubusercontent.com/57654726/120127567-45123680-c1f2-11eb-9561-a2f85962cfee.png)
[2021/05/31 09:06:40] root INFO: Architecture : 
[2021/05/31 09:06:40] root INFO:     Backbone : 
[2021/05/31 09:06:40] root INFO:         disable_se : True
[2021/05/31 09:06:40] root INFO:         model_name : large
[2021/05/31 09:06:40] root INFO:         name : MobileNetV3
[2021/05/31 09:06:40] root INFO:         scale : 0.5
[2021/05/31 09:06:40] root INFO:     Head : 
[2021/05/31 09:06:40] root INFO:         k : 50
[2021/05/31 09:06:40] root INFO:         name : DBHead
[2021/05/31 09:06:40] root INFO:     Neck : 
[2021/05/31 09:06:40] root INFO:         name : DBFPN
[2021/05/31 09:06:40] root INFO:         out_channels : 96
[2021/05/31 09:06:40] root INFO:     Transform : None
[2021/05/31 09:06:40] root INFO:     algorithm : DB
[2021/05/31 09:06:40] root INFO:     model_type : det
[2021/05/31 09:06:40] root INFO: Eval : 
[2021/05/31 09:06:40] root INFO:     dataset : 
[2021/05/31 09:06:40] root INFO:         data_dir : ./train_data/sfz/
[2021/05/31 09:06:40] root INFO:         label_file_list : ['./train_data/sfz/Label.txt']
[2021/05/31 09:06:40] root INFO:         name : SimpleDataSet
[2021/05/31 09:06:40] root INFO:         transforms : 
[2021/05/31 09:06:40] root INFO:             DecodeImage : 
[2021/05/31 09:06:40] root INFO:                 channel_first : False
[2021/05/31 09:06:40] root INFO:                 img_mode : BGR
[2021/05/31 09:06:40] root INFO:             DetLabelEncode : None
[2021/05/31 09:06:40] root INFO:             DetResizeForTest : None
[2021/05/31 09:06:40] root INFO:             NormalizeImage : 
[2021/05/31 09:06:40] root INFO:                 mean : [0.485, 0.456, 0.406]
[2021/05/31 09:06:40] root INFO:                 order : hwc
[2021/05/31 09:06:40] root INFO:                 scale : 1./255.
[2021/05/31 09:06:40] root INFO:                 std : [0.229, 0.224, 0.225]
[2021/05/31 09:06:40] root INFO:             ToCHWImage : None
[2021/05/31 09:06:40] root INFO:             KeepKeys : 
[2021/05/31 09:06:40] root INFO:                 keep_keys : ['image', 'shape', 'polys', 'ignore_tags']
[2021/05/31 09:06:40] root INFO:     loader : 
[2021/05/31 09:06:40] root INFO:         batch_size_per_card : 1
[2021/05/31 09:06:40] root INFO:         drop_last : False
[2021/05/31 09:06:40] root INFO:         num_workers : 0
[2021/05/31 09:06:40] root INFO:         shuffle : False
[2021/05/31 09:06:40] root INFO: Global : 
[2021/05/31 09:06:40] root INFO:     cal_metric_during_train : False
[2021/05/31 09:06:40] root INFO:     checkpoints : None
[2021/05/31 09:06:40] root INFO:     debug : False
[2021/05/31 09:06:40] root INFO:     distributed : False
[2021/05/31 09:06:40] root INFO:     epoch_num : 1200
[2021/05/31 09:06:40] root INFO:     eval_batch_step : [0, 2000]
[2021/05/31 09:06:40] root INFO:     infer_img : doc/imgs_en/img_10.jpg
[2021/05/31 09:06:40] root INFO:     log_smooth_window : 20
[2021/05/31 09:06:40] root INFO:     pretrained_model : ./pretrain_models/MobileNetV3_large_x0_5_pretrained
[2021/05/31 09:06:40] root INFO:     print_batch_step : 2
[2021/05/31 09:06:40] root INFO:     save_epoch_step : 1200
[2021/05/31 09:06:40] root INFO:     save_inference_dir : None
[2021/05/31 09:06:40] root INFO:     save_model_dir : ./output/ch_db_mv3/
[2021/05/31 09:06:40] root INFO:     save_res_path : ./output/det_db/predicts_db.txt
[2021/05/31 09:06:40] root INFO:     use_gpu : True
[2021/05/31 09:06:40] root INFO:     use_visualdl : False
[2021/05/31 09:06:40] root INFO: Loss : 
[2021/05/31 09:06:40] root INFO:     alpha : 5
[2021/05/31 09:06:40] root INFO:     balance_loss : True
[2021/05/31 09:06:40] root INFO:     beta : 10
[2021/05/31 09:06:40] root INFO:     main_loss_type : DiceLoss
[2021/05/31 09:06:40] root INFO:     name : DBLoss
[2021/05/31 09:06:40] root INFO:     ohem_ratio : 3
[2021/05/31 09:06:40] root INFO: Metric : 
[2021/05/31 09:06:40] root INFO:     main_indicator : hmean
[2021/05/31 09:06:40] root INFO:     name : DetMetric
[2021/05/31 09:06:40] root INFO: Optimizer : 
[2021/05/31 09:06:40] root INFO:     beta1 : 0.9
[2021/05/31 09:06:40] root INFO:     beta2 : 0.999
[2021/05/31 09:06:40] root INFO:     lr : 
[2021/05/31 09:06:40] root INFO:         learning_rate : 0.001
[2021/05/31 09:06:40] root INFO:         name : Cosine
[2021/05/31 09:06:40] root INFO:         warmup_epoch : 2
[2021/05/31 09:06:40] root INFO:     name : Adam
[2021/05/31 09:06:40] root INFO:     regularizer : 
[2021/05/31 09:06:40] root INFO:         factor : 0
[2021/05/31 09:06:40] root INFO:         name : L2
[2021/05/31 09:06:40] root INFO: PostProcess : 
[2021/05/31 09:06:40] root INFO:     box_thresh : 0.6
[2021/05/31 09:06:40] root INFO:     max_candidates : 1000
[2021/05/31 09:06:40] root INFO:     name : DBPostProcess
[2021/05/31 09:06:40] root INFO:     thresh : 0.3
[2021/05/31 09:06:40] root INFO:     unclip_ratio : 1.5
[2021/05/31 09:06:40] root INFO: Train : 
[2021/05/31 09:06:40] root INFO:     dataset : 
[2021/05/31 09:06:40] root INFO:         data_dir : ./train_data/sfz/
[2021/05/31 09:06:40] root INFO:         label_file_list : ['./train_data/sfz/Label.txt']
[2021/05/31 09:06:40] root INFO:         name : SimpleDataSet
[2021/05/31 09:06:40] root INFO:         ratio_list : [1.0]
[2021/05/31 09:06:40] root INFO:         transforms : 
[2021/05/31 09:06:40] root INFO:             DecodeImage : 
[2021/05/31 09:06:40] root INFO:                 channel_first : False
[2021/05/31 09:06:40] root INFO:                 img_mode : BGR
[2021/05/31 09:06:40] root INFO:             DetLabelEncode : None
[2021/05/31 09:06:40] root INFO:             IaaAugment : 
[2021/05/31 09:06:40] root INFO:                 augmenter_args : 
[2021/05/31 09:06:40] root INFO:                     args : 
[2021/05/31 09:06:40] root INFO:                         p : 0.5
[2021/05/31 09:06:40] root INFO:                     type : Fliplr
[2021/05/31 09:06:40] root INFO:                     args : 
[2021/05/31 09:06:40] root INFO:                         rotate : [-10, 10]
[2021/05/31 09:06:40] root INFO:                     type : Affine
[2021/05/31 09:06:40] root INFO:                     args : 
[2021/05/31 09:06:40] root INFO:                         size : [0.5, 3]
[2021/05/31 09:06:40] root INFO:                     type : Resize
[2021/05/31 09:06:40] root INFO:             EastRandomCropData : 
[2021/05/31 09:06:40] root INFO:                 keep_ratio : True
[2021/05/31 09:06:40] root INFO:                 max_tries : 50
[2021/05/31 09:06:40] root INFO:                 size : [960, 960]
[2021/05/31 09:06:40] root INFO:             MakeBorderMap : 
[2021/05/31 09:06:40] root INFO:                 shrink_ratio : 0.4
[2021/05/31 09:06:40] root INFO:                 thresh_max : 0.7
[2021/05/31 09:06:40] root INFO:                 thresh_min : 0.3
[2021/05/31 09:06:40] root INFO:             MakeShrinkMap : 
[2021/05/31 09:06:40] root INFO:                 min_text_size : 8
[2021/05/31 09:06:40] root INFO:                 shrink_ratio : 0.4
[2021/05/31 09:06:40] root INFO:             NormalizeImage : 
[2021/05/31 09:06:40] root INFO:                 mean : [0.485, 0.456, 0.406]
[2021/05/31 09:06:40] root INFO:                 order : hwc
[2021/05/31 09:06:40] root INFO:                 scale : 1./255.
[2021/05/31 09:06:40] root INFO:                 std : [0.229, 0.224, 0.225]
[2021/05/31 09:06:40] root INFO:             ToCHWImage : None
[2021/05/31 09:06:40] root INFO:             KeepKeys : 
[2021/05/31 09:06:40] root INFO:                 keep_keys : ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask']
[2021/05/31 09:06:40] root INFO:     loader : 
[2021/05/31 09:06:40] root INFO:         batch_size_per_card : 8
[2021/05/31 09:06:40] root INFO:         drop_last : False
[2021/05/31 09:06:40] root INFO:         num_workers : 0

**Comment by xialei2821212670** [comment]
![image](https://user-images.githubusercontent.com/57654726/120127926-60317600-c1f3-11eb-97ff-bf7fc35735e0.png)

![image](https://user-images.githubusercontent.com/57654726/120127934-66275700-c1f3-11eb-94ff-1abae40b16dc.png)
